opensearch_py_ml/ml_models/sparse_model.py

```diff
@@ -0,0 +1,113 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import json
+import os
+from zipfile import ZipFile
+
+import requests
+
+LICENSE_URL = "https://github.com/opensearch-project/opensearch-py-ml/raw/main/LICENSE"
```

Comment:
Let's put this in a common file : may be ml_commons_utils.py?

Reply:
Ok, add a utils file to store some common variable.